test/cookbooks/etcd_test/recipes/keys.rb

```diff
@@ -0,0 +1,18 @@
+etcd_service 'default' do
+  action [:create, :start]
+end
+
+chef_gem "etcd" do
+  compile_time true
+end
```

Comment:
Does the etcd gem have dependencies or C extensions? If not, I think we should vendor it the same way docker-api is vendored in the Docker cookbook